project/CodeCoverageSettings.scala

```diff
@@ -0,0 +1,21 @@
+import sbt.Setting
+import scoverage.ScoverageKeys.*
+
+object CodeCoverageSettings {
+  private val excludedPackages: Seq[String] = Seq(
+    "<empty>",
+    ".*definition.*",
+    "prod.*",
+    "live.*",
+    "testOnlyDoNotUseInAppConf.*",
+    "app.*",
+    "uk.gov.hmrc.BuildInfo"
+  )
```

Comment:
can we set these to what they originally were for this repo (<empty>;.*components.*;.*Routes.*;)

Reply:
Changes made. However, when I run the AC's locally, it's failing on test coverage:

[info] Statement coverage.: 92.41%
[info] Branch coverage....: 85.77%
[info] Coverage reports completed
[error] Coverage is below minimum [92.41% < 99.00%]: Stmt:Total
[error] java.lang.RuntimeException: Coverage minimum was not reached
[error]         at scoverage.CoverageMinimum$All.checkCoverage(CoverageMinimum.scala:76)
[error]         at scoverage.ScoverageSbtPlugin$.$anonfun$coverageReport0$1(ScoverageSbtPlugin.scala:240)
[error]         at scoverage.ScoverageSbtPlugin$.$anonfun$coverageReport0$1$adapted(ScoverageSbtPlugin.scala:211)
[error]         at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error]         at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:63)
[error]         at sbt.std.Transform$$anon$4.work(Transform.scala:69)
[error]         at sbt.Execute.$anonfun$submit$2(Execute.scala:283)
[error]         at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:24)
[error]         at sbt.Execute.work(Execute.scala:292)
[error]         at sbt.Execute.$anonfun$submit$1(Execute.scala:283)
[error]         at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error]         at sbt.CompletionService$$anon$2.call(CompletionService.scala:65)
[error]         at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error]         at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
[error]         at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[error]         at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[error]         at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[error]         at java.base/java.lang.Thread.run(Thread.java:829)
[error] (coverageReport) Coverage minimum was not reached


How should I handle this?